front-end/src/security_privacy/rsa/key/private.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIICXQIBAAKBgQCUCG8zpA99Ph7uMlpgfmRnTohRYn9jdMAtxXPK7+b+Dju5C1Mh
+cwG5vl4gU9/TKryzukPPpXNdsrFjx882ohwxf7SZoG8IJLtZqXZfP8hJazxjqoKE
+jPPe6XwG9HqWTqZy7U4FS+sotyts4DVAefCQ2QHKsA9Jn6MXPJ0Gm9dSyQIDAQAB
+AoGAdbDe+CePSPuCi83snJrgKGH/bvToYqIHyFwbW1SwSXv3b37ZcNaGoK4BF6KX
+aMOpkxqbnxLjTbMr5TCORdlMn7VNhv4sRZTKM3osOIO4V7ys5tZOLxiI4vsGmIoZ
+61HYrf/xQ6umAVqUDSpEMpcUd/dCriOSt2cfngiW9eEXqAECQQDgNMYnebWTGH7i
+uiLGEMJrzRokm51tNnzmuni4cu78nQv+/xpuq3Oxx+S0UPNMh3pXsFn1iNkth994
+Fz2ztIepAkEAqQZjgsTM7FsAfRPL+2IZMQQfLLs1EvclOXZYf/BHeVK1YfM880qR
+/xzSWaMlxtOSk2zZcsmZPH9sPXtgUDDmIQJAb+OmaBjjNq9o0zlVFUdT33yR6WcH
+n3MrzfGNQW+KOOMfE3i39jb2G+jIB/zGDpElyxElM9XBnGlMaqAemMolUQJBAJ8B
+V5OpN24RncAMY5Q0kKY/zmPyKNeSj/4iitawX0yOtftTKz9kIzvvDSMuC1ejqPvv
+JhCJ/hGffV/JNK27koECQQC0pUahase7AXSUcC3Rkbq08PpFmwqMLwQk8gyLL4FU
+qGOPa5KoQMLxqzK3PB7H4+NDr6ueRw1dJl/zWtMQP3Zo
+-----END RSA PRIVATE KEY-----
```

Comment:
I don't think storing the private key in the source code is a good idea.
Is it possible for the program to run without the private key being exposed publicly?

Reply:
I agree with @JoshuaBuditama

Reply:
> I don't think storing the private key in the source code is a good idea.
> Is it possible for the program to run without the private key being exposed publicly?

Thanks for feed back. I was just try to test and see it first. I will try to find way to store it privately.

Reply:
@cskeogh @JoshuaBuditama  do u guys think using .env file is a better idea?

Reply:
> @cskeogh @JoshuaBuditama do u guys think using .env file is a better idea?

The back-end public key will be sent to the front-end in #35 (not implemented yet). The back-end public key can be stored in the browser's localStorage.
The front-end private-key - don't store anywhere, memory only. It's not needed to be stored. If there is no front-end private key in memory, generate a new keypair.
The front-end public-key, store together with the upload table, for providing to the server.

Note: as you may now have multiple keypairs, make sure you store the front-end public-key with the corresponding data.